src/main/resources/com/google/jenkins/plugins/k8sengine/KubernetesEnginePublisher/config.jelly

```diff
@@ -7,9 +7,18 @@
     <f:entry field="clusterName" title="${%Cluster}">
         <f:textbox/>
     </f:entry>
-    <f:entry field="zone" title="${%Zone}">
-        <f:textbox/>
-    </f:entry>
+    <f:section title="${%Zone}">
+        <f:radioBlock name="entryMethod" title="Dropdown" value="dropdown" checked="true" inline="true">
```

Comment:
I've made the dropdown the first element, since it is the default. Note for the future: The meaning of "checked" is the element in the set of radioBlock elements with the same name that is selected first. Afterwords, user clicks set the entryMethod through the usual setter, and the rendering of which entryMethod is selected comes from the getter.